pkg/resources/pciDevice.go

```diff
@@ -44,6 +45,9 @@ func NewPciDevice(dev *ghw.PCIDevice, rFactory types.ResourceFactory, rc *types.
 		return nil, err
 	}
 
+	// Get NIC name. Ignoring the error because not all PCI Devices will be NICs
+	nicName, _ := utils.GetNicName(pciAddr)
```

Comment:
There could be different errors returned in by GetNicName function. I'm not sure we should ignore all of them